src/entity/user/User.ts

```diff
@@ -22,6 +22,12 @@ export type UserFlags = Partial<{
   trustScore: number;
 }>;
 
+export type UserSubscriptionFlags = Partial<{
```

Comment:
Since we need to redirect the user to paddle management page, shouldn't we store an id or something like that?

Reply:
If we need that yes, haven't got to the manage link yet (just asked for that on slack)
I mean we can store what we want, but wanted to keep the flags as lean as possible.
Do you think we should just store everything?

Reply:
not everything, but since we know we need to send them to the management link. I'd make sure we have everything we need to do that. I'm assuming it would need some sort of subscription id

Reply:
It's hard for me to determine what we need, just add as I see a need...
I'll add this flag as well, but maybe during my loom Sam can let us know if we miss important info.

We could also consider storing the whole object somewhere else just for reading ops later?

Reply:
I'm afraid the object may contain personal info that we don't need. let's just add subscription id for now because i think it will be good for debugging and analytics. the rest let's leave it as is